src/parser/AdminSentences.cpp

```diff
@@ -96,12 +102,14 @@ std::string CreateSpaceSentence::toString() const {
     return buf;
 }
 
+
 std::string DropSpaceSentence::toString() const {
```

Comment:
const std::string

Reply:
Never qualify with const when you return by value.